chore: Update language onboarding guidelines to include container runtime specs (#2633)
googleapis/librarian: doc/language-onboarding.md

so by including a field in the various response files outlined below. Additionally, any logs sent to stderr/stdout will
be surfaced to the CLI.

> The runtime of the language container for processing any command should not exceed 1 minute. This is not a strict requirement, as we understand that some libraries may take longer to process. However, long runtimes can adversely affect your ability to roll out emergency changes. While the CLI typically calls the container only for libraries with changes, a generator update could trigger a run for all your libraries, which would significantly delay emergency updates.

Reviewer: I would say that generating a straightforward library such as for Google Cloud Functions should probably take significantly less than a minute. The outliers will be:

- AI Platform and Compute for generation, as they're huge
- Anything big (whether handwritten or generated) for build - but particularly libraries with a lot of unit tests

* Create a docker file for your container [example](https://github.com/googleapis/google-cloud-go/blob/main/internal/librariangen/Dockerfile)
* Create a cloudbuild file [example](https://github.com/googleapis/google-cloud-go/blob/main/internal/librariangen/cloudbuild-exitgate.yaml) that uploads your image to us-central1-docker.pkg.dev/cloud-sdk-librarian-prod/images-dev

### Guidelines on Language Container Runtimes

> You should be able to run the generate and release command for an API such as Google Cloud Functions in less than a

Reviewer: ```suggestion
You should be able to run the `generate` or `release-init` commands for an API such as Google Cloud Functions in less than a
```

(Just to clarify that these are the names of container commands.)
